packages/vuetify/src/composables/virtual.ts

```diff
@@ -33,7 +33,7 @@ export function useVirtual <T> (props: VirtualProps, items: Ref<readonly T[]>) {
 
   const itemHeight = shallowRef(0)
   watchEffect(() => {
-    itemHeight.value = parseFloat(props.itemHeight || 0)
+    itemHeight.value = parseFloat(props.itemHeight || 16)
```

Comment:
I don't have a better solution but magic numbers are bad.

Reply:
https://github.com/vuetifyjs/vuetify/blob/f5b13c153262b8d4b5e12a23d62d574b2f36ec48/packages/vuetify/src/composables/virtual.ts#L44

Setting it here could cause problems with items that are actually <16px